src/native/managed/cdac/Microsoft.Diagnostics.DataContractReader.Contracts/Contracts/RuntimeTypeSystem_1.cs

```diff
@@ -843,7 +843,7 @@ private TargetPointer GetAddressOfSlot(TypeHandle typeHandle, uint slotNum)
             TargetPointer auxDataPtr = mt.AuxiliaryData;
             Data.MethodTableAuxiliaryData auxData = _target.ProcessedData.GetOrAdd<Data.MethodTableAuxiliaryData>(auxDataPtr);
             TargetPointer nonVirtualSlotsArray = auxDataPtr + (ulong)auxData.OffsetToNonVirtualSlots;
-            return nonVirtualSlotsArray - (1 + (slotNum - mt.NumVirtuals));
+            return nonVirtualSlotsArray - ((1 + (slotNum - mt.NumVirtuals)) * (ulong)_target.PointerSize);
```

Comment:
I'd like to see `(1 + (slotNum - mt.NumVirtuals))` in a separate named local that describes what it is, so the index calculation is separated from the machinery of turning an index into an address